lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -94,6 +94,9 @@ do {
   $err_cur_line = SlurpOutput $err_file $err_cur_line 'err'
 } while (!($registered_task.state -eq 3))
 
+del $out_file
```

Comment:
I have found that its helpful leaving these files for potential troubleshooting purposes. They get yanked in the beginning so no need to worry that the wrong data will get reaped.